datadog_checks_dev/datadog_checks/dev/plugin/tox.py

```diff
@@ -11,6 +11,7 @@
 STYLE_CHECK_ENV_NAME = 'style'
 STYLE_FORMATTER_ENV_NAME = 'format_style'
 STYLE_FLAG = 'dd_check_style'
+MYPY_ARGS = 'mypy_args'
```

Comment:
WDYTA doing a new dynamic env instead behind a flag like `dd_check_types`?

Reply:
As in… have a `MYPY_ARGS` env variable that would be used if `dd_check_types = true` in `tox.ini`?

```ini
[testenv]
dd_check_types = true
setenv =
    MYPY_ARGS=path/to/files/to/check.py --my-option
```

Reply:
Yes exactly

Reply:
I wasn't able to use an environment variable passed via `setenv` because those environment variables aren't set yet during `tox_configure()`.

As an alternative, I kept the idea of a flag and prefixed the `mypy_args` option with `dd_`:

```ini
[testenv]
dd_check_types = true
dd_mypy_args = --py2 --my-option /path/to/file/to/check.py /path/to/more/files
```

Note that I removed the `python_version = 2.7` option in `mypy.ini` (used to tell it "hey please make sure this code is 2.7-compatible), since it's global and we can't define it per-module or per-file, so that makes IDEs complain when using Python 3-only code (e.g. in `ddev`). Instead, I think we should enforce it on a per-integration basis using `--py2`, which is what I did here.